src/lib.rs

```diff
@@ -913,8 +913,8 @@ mod tests {
             assert_eq!(None, iter.next());
         }
 
-        assert_eq!(17, map["a"]);
-        assert_eq!(23, map["b"]);
+        assert_eq!(17, map[&"a"]);
+        assert_eq!(23, map[&"b"]);
```

Comment:
Really? It makes sense to me. It's a Map<&str, i32>. Seems to follow the &&str is necessary?